hcloud/instances.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/hetznercloud/hcloud-go/v2/hcloud"
 )
 
+const (
+	ProvidedBy = "instance.hetzner.cloud/provided-by"
```

Comment:
Do we have a documentation for how to create a new label? 

According to https://kubernetes.io/docs/reference/labels-annotations-taints, we might want to use: `node.hetzner.cloud/provided-by` ?

Correct me if I am wrong, but since we manage the whole lifecycle of the nodes, shouldn't we use `node.hetzner.cloud/managed-by` ?

Reply:
There is no documentation for defining new labels at the moment.

`instance.hetzner.cloud/` is the prefix used by the previous csi-driver label `instance.hetzner.cloud/is-root-server`, so the idea was to just stick with the same prefix.

> Correct me if I am wrong, but since we manage the whole lifecycle of the nodes, shouldn't we use node.hetzner.cloud/managed-by ?

We do not manage the whole lifecycle here. Creating nodes is out of scope for HCCM. Managed By is more the controller/application that creates the Kubernetes Object, so in this case it could be one of `kubelet` or `hcloud-cloud-controller-manager`.

---

Happy to hear other suggestions